test/unit/factorization/mc/test_compute_t_factor.cpp

```diff
@@ -237,12 +237,7 @@ std::vector<std::tuple<SizeType, SizeType, SizeType, SizeType, SizeType, GlobalT
 //
 // Which we expect to be the equal to the one computed previously.
 TYPED_TEST(ComputeTFactorTestMC, CorrectnessLocal) {
-  SizeType a_m, a_n, mb, nb, k;
-  GlobalTileIndex v_start;
-
-  for (auto config : configs) {
-    std::tie(a_m, a_n, mb, nb, k, v_start) = config;
-
+  for (auto& [a_m, a_n, mb, nb, k, v_start] : configs) {
```

Comment:
This can lead to a change in the configs vector.
If the parameter can be modified, please copy them, otherwise take a const reference.